diagnosis/getkuberneteslogs.sh

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash
 
+RED='\033[0;31m'    # For error
```

Comment:
DRY - Do not repeat yourself 

You can import the existing function with something like `source common.sh` (see other scrips for examples)